src/components/Layout/index.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <div class="max-w-lg min-h-screen mx-auto shadow-md">
+    <div class="bg-[#FEFEFE]">
```

Comment:
are you using tailwind css  ?

if you used tailwind css,

u can write `bg-white` for background white

suggestion :

https://tailwindcss.com/docs/background-color

Reply:
`#FEFEFE` is a very light gray color

Reply:
Yes i used tailwind css, and i write `#FEFEFE` for the very light gray color. Thankyou for the suggestion @rayfajars